controllers/tlspolicies_validator.go

```diff
@@ -99,6 +104,22 @@ func (t *TLSPoliciesValidator) isTargetRefsFound(topology *machinery.Topology, p
 	return nil
 }
 
+// isConflict Validates if there's already an older policy with the same target ref
+func (t *TLSPoliciesValidator) isConflict(policies []machinery.Policy, p *kuadrantv1.TLSPolicy) error {
+	conflictingP, ok := lo.Find(policies, func(item machinery.Policy) bool {
+		conflictTLSPolicy := item.(*kuadrantv1.TLSPolicy)
+		return p != conflictTLSPolicy && conflictTLSPolicy.DeletionTimestamp == nil &&
+			conflictTLSPolicy.CreationTimestamp.Before(&p.CreationTimestamp) &&
+			reflect.DeepEqual(conflictTLSPolicy.GetTargetRefs(), p.GetTargetRefs())
```

Comment:
ok this I expected. would it make sense above just to check the names are not equal as that couldn't happen right?

Reply:
Hmm, I went with deep equal to also cover the case of same gateway but different section. This would be allowed. It's only when the target is exactly the same that we report that it's conflcted by another policy 🤔

Reply:
I think I wasn't clear. This is fine

Reply:
I _think_ this is fine, but i was wondering if already accepted policies should be explicitly checked, and not just the timestamps. One policy per target should only ever be accepted, if one is already accepted for this target, all others should conflict regardless of when they were created.